src/components/dashboard/main-content/course-enrollments/data/tests/hooks.test.jsx

```diff
@@ -589,11 +589,15 @@ describe('useCourseUpgradeData', () => {
 });
 
 describe('useContentAssignments', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
   const mockRedeemableLearnerCreditPolicies = emptyRedeemableLearnerCreditPolicies;
   const mockSubsidyExpirationDateStr = dayjs().add(ENROLL_BY_DATE_WARNING_THRESHOLD_DAYS + 1, 'days').toISOString();
   const mockAssignmentConfigurationId = 'test-assignment-configuration-id';
   const mockAssignment = {
     contentKey: 'edX+DemoX',
+    parentContentKey: 'course-v1:Best+course+2T2025',
```

Comment:
If the assignment's `contentKey` is associated with a top-level course key, `parentContentKey` would be `null` (there is no parent to a course). `parentContentKey` should reflect a top-level course key only when the assignment's `contentKey` represents a course run.

As such, the `isAssignedCourseRun: true` below is a bit misleading, given that the assignment's `contentKey` is actually for a top-level course, not a course run.